scripts/load-vars.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+curl -g https://us-central1-clarity-design-system.cloudfunctions.net/applitools -o .env
```

Comment:
I know there isn't much we can do but 🙁

Reply:
Its only meant to run on CI and this is the easiest script to write without writing a longer node script.